RPGSystem/Form1.cs

```diff
@@ -16,5 +16,17 @@ public Form1()
         {
```

Comment:
Please rename Form1 to something meaningful

Reply:
File was deleted